### ISSUE TITLE
Amazon Glacier support?

### ISSUE THREAD

**Issue by ghost** [alexkravets/heroku-mongo-backup#12]
Not an issue, but this would be a very helpful feature.

**Comment by alexkravets** [comment]
Yeah, seems to be like a good idea. Not sure if there will be a time frame
for this. But everyone's welcome to commit the feature support.
